### PR TITLE
Types in products

### PR DESCRIPTION
**Pull request by tobycm**

### CODE REVIEW

**Comment by tobycm** [comment]
:b

**Comment by tobycm** [comment]
image
any idea? :b

**Comment by pdt1806** [comment]
is there any way to expand the height and limit the width of each ?

**Comment by pdt1806** [comment]
> is there any way to expand the height and limit the width of each ?

try [this](https://mantine.dev/core/segmented-control/#react-node-as-label)

**Comment by pdt1806** [comment]
> > is there any way to expand the height and limit the width of each ?
> 
> try [this](https://mantine.dev/core/segmented-control/#react-node-as-label)

didn't work as expected, try another method

File: src/pages/Gallery/index.tsx
Comment (pdt1806) [inline src/pages/Gallery/index.tsx:1]: bruh

**Review by pdt1806 — APPROVED** [review]
looks good to me

**Comment by tobycm** [comment]
> looks good to me

bro is turning into a generic reviewer xddddd

**Pull request merged**